config/schedule.rb

```diff
@@ -15,3 +15,9 @@
   runner 'WorkerHealthChecker.check'
   runner 'WorkerHealthChecker.enqueue_dummy_jobs'
 end
+
+mail_batch = Whenever.seconds(Figaro.env.usps_mail_batch_hours.to_i, :hours)
+
+every mail_batch, roles: [:job_creator] do
+  runner 'UspsUploader.run'
```

Comment:
Yes, great catch. I went back and forth on that a million times knowing I'd make this mistake....